lib/newrelic.js

```diff
@@ -113,7 +113,7 @@ class NewRelic {
     /**
      * Call this when the action activation finishes successfully.
      */
-    async activationFinished() {
+    activationFinished() {
```

Comment:
nice to have: can you remove all the `await`'s in the tests for these? https://github.com/adobe/node-openwhisk-newrelic/blob/50fbd58c5b2c9e054dac145b36d1311544626b2b/test/newrelic.test.js#L165

Reply:
also we call `await` on this in commons:
https://github.com/adobe/asset-compute-commons/blob/master/lib/wrap.js#L70
.what this method supported too be async but we didn't return on: 
```
return clearTimeout(this.actionTimeoutHandlerId);
```

Reply:
One library at a time, @jdelbick. Let me start releasing this. 
Also await/async is "only" syntactic sugar, await a sync function shouldn't interfere.